storage/imx6ull-flashnor/Makefile

```diff
@@ -4,10 +4,18 @@
 # Copyright 2021 Phoenix Systems
 #
 
-NAME := imx6ull-flashnor
-LOCAL_SRCS := flashnor.c flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor/nor.c
-LOCAL_HEADERS := flashnor-ecspi.h flashnor-qspi.h flashnor-drv.h nor/nor.h
+
+NAME := libflashnordrv-imx6ull
+LOCAL_SRCS := flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor.c
+LOCAL_HEADERS := imx6ull-flashnor-ecspi.h imx6ull-flashnor-qspi.h imx6ull-flashnor-drv.h imx6ull-nor.h
```

Comment:
Are these all files external API?
Maybe put them as
`include/imx6ull-flashnor/ecspi.h` ... 

And use as `#include <imx6ull-flashnor/ecspi.h>`?

Edit:
As this lib is being used only inside the same dir (component) I would opt not to make it an explicit library (don't install headers, use objects directly - make would ensure they would be built only once and in correct order).

I'm thinking about something along the lines (not tested):

```make
IMX6ULL_FLASHNOR_COMMON_SRCS := flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor.c

NAME := imx6ull-flashnor
LOCAL_SRCS := $(IMX6ULL_FLASHNOR_COMMON_SRCS) flashnor-srv.c
DEP_LIBS := libimx6ull-ecspi libimx6ull-qspi
LIBS := libmeterfs-storage libmeterfs libstorage libjffs2 libmtd
LOCAL_INSTALL_PATH := /sbin
include $(binary.mk)

NAME := flashnor-test
LOCAL_SRCS := $(IMX6ULL_FLASHNOR_COMMON_SRCS) flashnor-test.c
DEP_LIBS := libimx6ull-ecspi libimx6ull-qspi
include $(binary.mk)
```